lib/auth/v4/validateInputs.js

```diff
@@ -41,7 +41,7 @@ function validateCredentials(credentials, timestamp, log) {
            { scopeDate, timestampDate });
         return errors.RequestTimeTooSkewed;
     }
-    if (service !== 's3' && service !== 'iam') {
+    if (service !== 's3' && service !== 'iam' && service !== 'ring') {
         log.warn('service in credentials is not s3', { service });
```

Comment:
You can amend this log line as well to `service in credentials is not one of s3/iam/ring`